tests/plugin/test_http_proxy_plugins_with_tls_interception.py

```diff
@@ -178,32 +176,59 @@ def send(raw: bytes) -> int:
         )
         self.assertFalse(self.protocol_handler.work.has_buffer())
 
-    def test_modify_post_data_plugin(self) -> None:
+        #
```

Comment:
?

Reply:
Thank you, yes a residual.  I had to copy lines above `#` out of `setUp` because I couldn't make async fixture work correctly.  So I added a marker for later revisit, could have been a TODO indeed.

Reply:
Created https://github.com/abhinavsingh/proxy.py/issues/775 to address various `pytest` related migration changes.  Will address this `#` once as part of `fixtures` setup.